frontend/src/components/UserCard.js

```diff
@@ -30,7 +30,7 @@ export default class UserCard extends Component {
 
     const href = `https://opencollective.com/${user.username}`;
 
-    const roleLabel = formatAnchor(user.tier || user.role);
+    const roleLabel = formatAnchor(user.role || user.tier);
```

Comment:
but I'd argue that the tier should always take precedence. This looks like the wrong fix. Otherwise "sponsor" will never show up (it will show backer which is the role)